ui/components/component-library/header-base/header-base.stories.tsx

```diff
@@ -0,0 +1,340 @@
+        }
+        {...args}
+      >
+        <Text
+          variant={TextVariant.headingSm}
+          textAlign={TEXT_ALIGN.CENTER}
+          backgroundColor={BackgroundColor.primaryAlternative}
+        >
+          Title is sentence case no period
+        </Text>
+      </HeaderBase>
+    </Box>
+    <Text>children, startAccessory, and endAccessory assigned </Text>
+    <Box backgroundColor={BackgroundColor.warningAlternative}>
+      <HeaderBase
+        marginBottom={4}
+        startAccessory={
+          <Button
+            backgroundColor={BackgroundColor.successAlternative}
+            style={{ whiteSpace: 'nowrap' }}
+            size={BUTTON_SIZES.SM}
+          >
+            Unlock Now
+          </Button>
+        }
+        endAccessory={
+          <ButtonIcon
+            backgroundColor={BackgroundColor.goerli}
+            size={BUTTON_ICON_SIZES.SM}
+            iconName={ICON_NAMES.CLOSE}
+            ariaLabel="close"
+          />
+        }
+        {...args}
+      >
+        <Text
+          variant={TextVariant.headingSm}
+          textAlign={TEXT_ALIGN.CENTER}
+          backgroundColor={BackgroundColor.primaryAlternative}
+        >
+          Title is sentence case no period
+        </Text>
+      </HeaderBase>
+    </Box>
+    <Text>
+      children, startAccessory, and endAccessory assigned with prop alignItems=
+      {AlignItems.center} passed at HeaderBase
+    </Text>
+    <Box backgroundColor={BackgroundColor.warningAlternative}>
+      <HeaderBase
+        marginBottom={4}
+        alignItems={AlignItems.center}
+        startAccessory={
+          <ButtonIcon
+            backgroundColor={BackgroundColor.successAlternative}
+            size={BUTTON_ICON_SIZES.SM}
+            iconName={ICON_NAMES.CLOSE}
+            ariaLabel="close"
+          />
+        }
+        endAccessory={
+          <Button
+            backgroundColor={BackgroundColor.goerli}
+            size={BUTTON_SIZES.SM}
+          >
+            Download
+          </Button>
+        }
+        {...args}
+      >
+        <Text
+          variant={TextVariant.headingSm}
+          textAlign={TEXT_ALIGN.CENTER}
+          backgroundColor={BackgroundColor.primaryAlternative}
+        >
+          Title is sentence case no period
+        </Text>
+      </HeaderBase>
+    </Box>
+    <Text>startAccessory and endAccessory assigned </Text>
+    <Box backgroundColor={BackgroundColor.warningAlternative}>
+      <HeaderBase
+        marginBottom={4}
+        startAccessory={
+          <Button
+            backgroundColor={BackgroundColor.successAlternative}
+            size={BUTTON_SIZES.SM}
+          >
+            Unlock
+          </Button>
+        }
+        endAccessory={
+          <ButtonIcon
+            backgroundColor={BackgroundColor.goerli}
+            size={BUTTON_ICON_SIZES.SM}
+            iconName={ICON_NAMES.CLOSE}
+            ariaLabel="close"
+          />
+        }
+        {...args}
+      ></HeaderBase>
```

Comment:
Nit: Could use self closing tag here
```suggestion
      />
```